### DISCUSSION TITLE
Cannot see the deployed model in cvat's model tab

### DISCUSSION THREAD

**Discussion by hunterchenghx** [cvat-ai/cvat#3423] [Q&A] [answered]
Excuse me.
I followed your instructions(https://github.com/openvinotoolkit/cvat/blob/49e4875dad6d2b61d306078f25d05e69dab97f50/site/content/en/docs/manual/advanced/serverless-tutorial.md) to build up the cvat with severless function.
![image](https://user-images.githubusercontent.com/45347759/125922986-10cd5445-b4be-4973-9e5e-ce2946da66e5.png)
When I deploy the function, although there are some warnings during the image pulling
![image](https://user-images.githubusercontent.com/45347759/125923342-dd75eb6e-92fe-42c3-99ac-85af097574dc.png)
But finally, it seems to be working. However, though I refreshed the page several times, there are still nothing in the model tabs. Could you tell me how to solve it?
![image](https://user-images.githubusercontent.com/45347759/125923741-fc9d6d42-609d-47f0-a547-3d8119c39ac1.png)

**Comment by nmanovic** [comment, marked as answer]
@hunterchenghx , try to update nuclio till 1.6.5 version (both dashboard and nuctl). I will prepare a pull request soon. You have PORT equals to 0.

**Reply by hunterchenghx** [reply]
Thank you so much for your reply. @nmanovic 
I changed my nuclio to 1.6.5(I also tried 1.6.12). This time, the node port is not 0. But I still cannot see anything in localhost:8080 model tab. Is there anything related to this node port?
![image](https://user-images.githubusercontent.com/45347759/126097448-0a5857c8-f6ad-4ca9-81fe-73101e6e00fc.png)

**Reply by efcy** [reply]
Did you set up cvat locally or via docker. I have the same problem with local dev setup. 

If your have the docker setup for cvat try reloading the page. I had to clear the cache then load the page again and then the models showed up.

**Reply by hunterchenghx** [reply]
> Did you set up cvat locally or via docker. I have the same problem with local dev setup.
> 
> If your have the docker setup for cvat try reloading the page. I had to clear the cache then load the page again and then the models showed up.

Yes, I did. The project name created by nuctl must be exactly "cvat", not other self-created names. After revise that, I could see the models.

**Reply by efcy** [reply]
Alright. I guess this can be set to answered then.
